src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,311 @@
+<pipeline_name>/
+    sub-<participant_label>/
+        anat/
+            <source_keywords>[_hemi-{L|R}][_space-<space>]_dseg.{label.gii|dlabel.nii}
+```
+
+The REQUIRED extension for GIFTI parcellations is `.label.gii`. The hemi tag is
+REQUIRED for GIFTI files. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_hemi-L_dparc.label.gii
+            sub-001_hemi-R_dparc.label.gii
+```
+
+The REQUIRED extension for CIFTI parcellations is `.dlabel.nii`. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_dparc.dlabel.nii
+            sub-001_dparc.dlabel.nii
+```
+
+### Anatomical Labels
+
+BIDS supplies a standard, generic label-index dictionary, defined in the table
+below, that contains common tissue classes and can be used to map segmentations
+(and parcellations) between lookup tables.
+
+| Integer value | Description             | Abbreviation (label) |
+| ------------- | ----------------------- | -------------------- |
+| 0             | Background              | BG                   |
+| 1             | Grey Matter             | GM                   |
+| 2             | White Matter            | WM                   |
+| 3             | Cerebrospinal Fluid     | CSF                  |
+| 4             | Grey and White Matter   | GWM                  |
+| 5             | Bone                    | B                    |
+| 6             | Soft Tissue             | ST                   |
+| 7             | Non-brain               | NB                   |
+| 8             | Lesion                  | L                    |
+| 9             | Cortical Grey Matter    | CGM                  |
+| 10            | Subcortical Grey Matter | SCGM                 |
+| 11            | Brainstem               | BS                   |
+| 12            | Cerebellum              | CBM                  |
+
+These definitions can be overridden (or added to) by providing custom labels in
+a sidecar `<matches>.tsv` file, in which `<matches>` corresponds to segmentation
+filename.
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_dseg.nii.gz
+            sub-001_space-orig_dseg.tsv
+```
+
+Definitions can also be specified with a top-level dseg.tsv, which propagates to
+segmentations in relative subdirectories.
+
+Example:
+
+```Text
+pipeline/
+    dseg.tsv
+    sub-001/
+        anat/
+            sub-001_space-orig_dseg.nii.gz
+```
+
+These tsv lookup tables should contain the following columns:
+
+| Column name | Description                                                             |
+| ----------- | ----------------------------------------------------------------------- |
+| index       | REQUIRED. The label integer index                                       |
+| name        | REQUIRED. The unique label name                                         |
+| abbr        | OPTIONAL. The unique label abbreviation                                 |
```

Comment:
https://github.com/bids-standard/bids-specification/pull/207/files#r296711258

Reply:
I'm okay with this. Does anybody object?

@ahoopes and @edickie I believe this was your contribution...

```suggestion
| abbreviation | OPTIONAL. The unique label abbreviation                                 |
```